keyboards/flx/lodestone/info.json

```
@@ -0,0 +1,84 @@
+"keyboard_name": "lodestone", 
```

Your JSON file is malformed.

```suggestion
{
    "keyboard_name": "lodestone",
```